OMCompiler/SimulationRuntime/OMSICpp/runtime/src/Core/Modelica/CMakeLists.txt

```diff
@@ -181,7 +181,13 @@ ENDIF()
 
 SET(Boost_LIBRARIES_NEW)
 FOREACH(lib ${Boost_LIBRARIES_})
-  GET_FILENAME_COMPONENT(libNew "${lib}" NAME_WE)
+  # Convert "Boost:SomeCrazyLib" to "-lboost_SomeCrazyLib" for ModelicaConfig_gcc.inc
+  GET_TARGET_PROPERTY(libNew "${lib}" LOCATION)
```

Comment:
This will generate a bunch of CMake warnings because `LOCATION` is non existing on older CMake.
```bash
CMake Warning (dev) at runtime/src/Core/Modelica/CMakeLists.txt:185 (get_target_property):
  Policy CMP0045 is not set: Error on non-existent target in
  get_target_property.  Run "cmake --help-policy CMP0045" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  get_target_property() called with non-existent target
  "/usr/lib/x86_64-linux-gnu/libboost_serialization.so".
This warning is for project developers.  Use -Wno-dev to suppress it.
```

Reply:
I guess we could do an ifdef on the cmake version.

Reply:
We could use: `if(${CMAKE_VERSION} VERSION_GREATER "3.16.0")`.

Reply:
Sound good. And otherwise the Cpp will issue a giant wall of CMake warnings anyway.

Reply:
ok, will do that.